azure-pipelines.yml

```
@@ -0,0 +1,31 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+- main
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+
+- task: CmdLine@2
+  displayName: 'Install sauceCtl'
```

`saucectl` would be better?

```suggestion
  displayName: 'Install saucectl'
```